client/app/components/pages.js

```diff
@@ -10,7 +10,8 @@ import topbar from "./_topbar/topbar";
 import landingPage from "./landingPage/landingPage";
 import notifications from "./notifications/notifications";
 import groupInfo from "./groupInfo/groupInfo";
-import groupList from "./groupList/groupList";
+import groupList from "./_groupList/groupList";
```

Comment:
groupList should not be imported here, but where it is used (e.g. joinGroupList)